test/performance/range/views/view_comparison_seq_read_benchmark.cpp

```diff
@@ -0,0 +1,237 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <list>
+#include <vector>
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/alphabet/all.hpp>
+#include <seqan3/range/views/all.hpp>
+#include <seqan3/test/performance/sequence_generator.hpp>
+#include <seqan3/test/seqan2.hpp>
+
+enum tag
+{
+    baseline,
+    slice,
+    comp,
```

Comment:
```suggestion
    complement,
```

Reply:
To illustrate my point from above: we have two entities in SeqAn3 that are called "complement": `seqan3::complement` and `seqan3::views::complement`. Calling the local entitiy by the same name *reduces* readability of the code.